wipp-backend-data/src/main/java/gov/nist/itl/ssd/wipp/backend/data/tensorboard/TensorBoardLogsController.java

```diff
@@ -0,0 +1,139 @@
+ * copy and distribute such modifications or works. Modified works should carry
+ * a notice stating that you changed the software and should note the date and
+ * nature of any such change. Please explicitly acknowledge the National
+ * Institute of Standards and Technology as the source of the software.
+ *
+ * NIST-developed software is expressly provided "AS IS." NIST MAKES NO
+ * WARRANTY OF ANY KIND, EXPRESS, IMPLIED, IN FACT, OR ARISING BY OPERATION OF
+ * LAW, INCLUDING, WITHOUT LIMITATION, THE IMPLIED WARRANTY OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT, AND DATA ACCURACY. NIST
+ * NEITHER REPRESENTS NOR WARRANTS THAT THE OPERATION OF THE SOFTWARE WILL BE
+ * UNINTERRUPTED OR ERROR-FREE, OR THAT ANY DEFECTS WILL BE CORRECTED. NIST
+ * DOES NOT WARRANT OR MAKE ANY REPRESENTATIONS REGARDING THE USE OF THE
+ * SOFTWARE OR THE RESULTS THEREOF, INCLUDING BUT NOT LIMITED TO THE
+ * CORRECTNESS, ACCURACY, RELIABILITY, OR USEFULNESS OF THE SOFTWARE.
+ *
+ * You are solely responsible for determining the appropriateness of using and
+ * distributing the software and you assume all risks associated with its use,
+ * including but not limited to the risks and costs of program errors,
+ * compliance with applicable laws, damage to or loss of data, programs or
+ * equipment, and the unavailability or interruption of operation. This
+ * software is not intended to be used in any situation where a failure could
+ * cause risk of injury or damage to property. The software developed by NIST
+ * employees is not subject to copyright protection within the United States.
+ */
+package gov.nist.itl.ssd.wipp.backend.data.tensorboard;
+
+import gov.nist.itl.ssd.wipp.backend.core.CoreConfig;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.*;
+import java.net.URL;
+import java.nio.channels.Channels;
+import java.nio.channels.ReadableByteChannel;
+import java.util.*;
+
+/**
+ * TensorBoard controller
+ *
+ * @author Antoine Meyer <antoine.meyer at nist.gov>
+ */
+@RestController
+@Tag(name="TensorboardLogs Entity")
+@RequestMapping(CoreConfig.BASE_URI + "/tensorboardLogs/{id}")
+public class TensorBoardLogsController {
+
+    @Autowired
+    CoreConfig config;
+
+    @Autowired
+    TensorboardLogsRepository tensorboardLogsRepository;
+
+    /**
+     * Download TensorFlow results (through TensorBoard instance) as CSV file
+     * @param id
+     * @throws IOException
+     */
+    @RequestMapping(value = "export/csv", method = RequestMethod.GET)
```

Comment:
a `@PreAuthorize` should be added to the two methods to enable access control (checkout for example the AiModel download controller)